docs/getting_started.md

```diff
@@ -4,21 +4,23 @@ We want to make it as easy as possible for you to start engaging with Suzieq. We
 
 ### As a Docker Container
 
-To get started seeing the kind of analysis Suzieq provides today, we have a demo container that has data in including the the iamge.
-
+To get started seeing the kind of analysis Suzieq provides today, we have a demo container that has data in including the the image.
+```
     docker run -it -p 8501:8501 --name suzieq netenglabs/suzieq-demo
```

Comment:
If you look at the original and this, this is broken. It looks like a comment in C++